### PR TITLE
More Preview 8 APIs

### PR DESCRIPTION
**Pull request by gewarren**
Follow-up to #4744 to add missing APIs in extensions assemblies. For example, there are two versions of microsoft.extensions.logging.console.dll (16 KB and 45 KB) in the "microsoft.aspnetcore.app" and "extensions" folders, respectively. The smaller version was used in #4744. This PR uses the version from the "extensions" folder in the assemblies drop location.

### CODE REVIEW

File: xml/FrameworksIndex/net-5.0.xml [hunk 1]
```diff
@@ -44556,56 +44555,6 @@
       <Member Id="F:System.Runtime.InteropServices.ComTypes.VARKIND.VAR_STATIC" />
     </Type>
   </Namespace>
-  <Namespace Name="System.Runtime.InteropServices.WindowsRuntime">
```
Comment (gewarren) [inline xml/FrameworksIndex/net-5.0.xml:44559]: Not sure why these got deleted.

**Comment by safern** [comment]
So where was the smaller version used? I will have to look at what happened here.

**Comment by gewarren** [comment]
> So where was the smaller version used? I will have to look at what happened here.

I didn't use the smaller versions. In fact I didn't use any of the 33 files in the "microsoft.aspnetcore.app" folder since they all existed in the "extensions" folder. For preview 7, some of the assemblies weren't in the "extensions" folder that you gave us, so we copied them from "microsoft.aspnetcore.app".

**Comment by safern** [comment]
Really? I would think that extensions has more assets than the aspnet folder.

**Comment by gewarren** [comment]
Sorry, I meant any of the 33 files in this list:

Microsoft.Extensions.Caching.Abstractions 
Microsoft.Extensions.Caching.Memory 
Microsoft.Extensions.Configuration 
Microsoft.Extensions.Configuration.Abstractions 
Microsoft.Extensions.Configuration.Binder 
Microsoft.Extensions.Configuration.CommandLine 
Microsoft.Extensions.Configuration.EnvironmentVariables 
Microsoft.Extensions.Configuration.FileExtensions 
Microsoft.Extensions.Configuration.Ini 
Microsoft.Extensions.Configuration.Json 
Microsoft.Extensions.Configuration.UserSecrets 
Microsoft.Extensions.Configuration.Xml 
Microsoft.Extensions.DependencyInjection 
Microsoft.Extensions.DependencyInjection.Abstractions 
Microsoft.Extensions.FileProviders.Abstractions 
Microsoft.Extensions.FileProviders.Composite 
Microsoft.Extensions.FileProviders.Physical 
Microsoft.Extensions.FileSystemGlobbing 
Microsoft.Extensions.Hosting 
Microsoft.Extensions.Hosting.Abstractions 
Microsoft.Extensions.Http 
Microsoft.Extensions.Logging 
Microsoft.Extensions.Logging.Abstractions 
Microsoft.Extensions.Logging.Configuration 
Microsoft.Extensions.Logging.Console 
Microsoft.Extensions.Logging.Debug 
Microsoft.Extensions.Logging.EventLog 
Microsoft.Extensions.Logging.EventSource 
Microsoft.Extensions.Logging.Testing 
Microsoft.Extensions.Logging.TraceSource 
Microsoft.Extensions.Options 
Microsoft.Extensions.Options.ConfigurationExtensions 
Microsoft.Extensions.Options.DataAnnotations 
Microsoft.Extensions.Primitives

File: xml/FrameworksIndex/net-5.0.xml (hunk 1 above)
Comment (gewarren) [reply xml/FrameworksIndex/net-5.0.xml:44559]: It looks like this is expected. For example, EventRegistrationTokenTable was removed in [this commit](https://github.com/dotnet/runtime/pull/36715/commits/77a50e79ae2f1b0d93689e02f39e3912cdc20157?file-filters%5B%5D=).

**Comment by gewarren** [comment]
This looks good to me. Could someone approve it?

**Review by safern — APPROVED** [review]

**Pull request merged**